src/electionguard/guardian.py

```diff
@@ -154,6 +154,13 @@ def __init__(
         sequence_order: int,
         number_of_guardians: int,
         quorum: int,
+        election_keys: ElectionKeyPair = None,
+        backups_to_share: Dict[GuardianId, ElectionPartialKeyBackup] = None,
+        election_public_keys: Dict[GuardianId, ElectionPublicKey] = None,
+        partial_key_backups: Dict[GuardianId, ElectionPartialKeyBackup] = None,
+        election_partial_key_verifications: Dict[
+            GuardianId, ElectionPartialKeyVerification
+        ] = None,
         nonce_seed: Optional[ElementModQ] = None,
```

Comment:
The nonce and the election_keys are in direct conflict with each other.

Reply:
That is true.  Users would need to either not supply the final optional argument and/or use named parameters.  If that is undesirable, I would need guidance on how to fix, since as far as I know Python doesn't support constructor overloading, which is how I would address it in a different language.